Makefile

```diff
@@ -53,3 +53,25 @@ image/build/push: image/build image/push
 test/unit:
 	@echo Running tests:
 	go test -v -race -cover ./pkg/...
+
+.PHONY: minikube/prepare/local
```

Comment:
I'll refactor this, it isn't actually anything minikube specific , same on L# 66 , 70